stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>Not a criticism, just an observation, but it's interesting that - if I'm understanding - you sort the teams by total wins, rather than proportion of wins, so they're not ordered (necessarily) by the strength of the teams. Is this to make the graph clearer?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>